community_server/src/Model/Table/TransactionsTable.php

```diff
@@ -256,7 +256,7 @@ public function listTransactionsHumanReadable($stateUserTransactions, array $use
                 $state_balance->record_date = $su_transaction->balance_date;
                 $final_transactions[] = [
                     'type' => 'decay',
-                    'balance' => floatval(-intval($su_transaction->balance - $state_balance->decay)) / 10000,
+                    'balance' => Number::format(floatval(-intval($su_transaction->balance - $state_balance->decay)) / 10000.0, ['precision' => 2]),
```

Comment:
Same here, please do not duplicate code, reuse it

Reply:
resolved with commit b10886a